### ISSUE TITLE
Gameboy Color Support?

### ISSUE THREAD

**Issue by WingofaGriffin** [Rodrigodd/gameroy#5]
Haven't seen any explicit mention of this, but are there plans to support GBC in the future?

**Comment by Rodrigodd** [comment]
No. When I started this project, my goal was to go as deep as possible in the parts that I want to explore (a nice GUI, maximum accuracy, android support, JIT compilation (planned), etc.), so I intentionally limit the emulator scope to only support DMG.

**Comment by Rodrigodd** [comment]
Thinking twice, I will keep this open. Some day, I or someone else may be interested in adding CGB support.
